client/src/components/TaskList/TaskList.js

```diff
@@ -56,28 +52,33 @@ export default function TaskList() {
         //Swap contents
         setContents((prevContents) => {
             const newContents = ObjArrayCopy(prevContents);
-
             const removedItem = { ...prevContents[result.source.index] };
             newContents.splice(result.source.index, 1);
             newContents.splice(result.destination.index, 0, removedItem);
 
-            //Map or foreach does not work to set timerIDStates.current. Explicit set required
-            timerIDStates.current.forEach((item, index) => {
-                clearInterval(item);
-                timerIDStates.current[index] = 0;
-            });
-
             return newContents;
         });
 
         //Delay setting dragging to false to allow drop animation to complete
         setTimeout(() => {
             setDragging(false);
         }, 100);
-    }, [timerIDStates]);
+    };
+
+    function dragStartHandler() {
+        //Set dragging which disables animation on circular progress and fade in/out of play/pause
+        setDragging(true);
+
+        //Disable all timers on pickup of any item
+        //Map or foreach does not work to set timerIDStates.current. Explicit set required
+        timerIDStates.current.forEach((item, index) => {
```

Comment:
Shouldn't this be put under dragEndHandler? I rmb previously the circular progress will still move even when user is dragging. The new implementation will stop it from moving, is that intended?